definitions/npm/@react-navigation/core_v3.x.x/flow_v0.60.x-/core_v3.x.x.js

```diff
@@ -0,0 +1,745 @@
+// @flow
+
+declare module '@react-navigation/core' {
+  /**
+   * First, a bunch of things we would love to import but instead must
+   * reconstruct (mostly copy-pasted).
+   */
+
+  // This is a bastardization of the true StyleObj type located in
+  // react-native/Libraries/StyleSheet/StyleSheetTypes. We unfortunately can't
+  // import that here, and it's too lengthy (and consequently too brittle) to
+  // copy-paste here either.
+  declare type StyleObj =
+    | null
+    | void
+    | number
+    | false
+    | ''
+    | $ReadOnlyArray<StyleObj>
+    | { [name: string]: any };
+  declare type ViewStyleProp = StyleObj;
+  declare type TextStyleProp = StyleObj;
+  declare type AnimatedTextStyleProp = StyleObj;
+
+  // This one is too large to copy. Actual definition is in
+  // react-native/Libraries/Animated/src/nodes/AnimatedValue.js
+  declare type AnimatedValue = Object;
+
+  declare type HeaderForceInset = {
+    horizontal?: string,
+    vertical?: string,
+    left?: string,
+    right?: string,
+    top?: string,
+    bottom?: string,
+  };
```

Comment:
We ask to use exact objects now on, if possible.

Reply:
Good call - this can definitely be refined a bit.